example/node/index.js

```diff
@@ -88,7 +88,7 @@ async function run() {
 
```

Comment:
WmsLayer should be also changed (couldn't put comment on exact line)
 
const layer = new WmsLayer({ baseUrl, layerId, title: 'My title', description: 'My desc' });

Reply:
Nice catch, thank you! Fixed.

I have checked now if linter would help protect against such errors, but couldn't make it issue a warning when the number of parameters was wrong, so I didn't enable it. I did however remove unused `OrbitDirection` in the process.